src/Parsers/ASTQueryWithOutput.h

```diff
@@ -18,13 +18,11 @@ class ASTQueryWithOutput : public IAST
     bool is_into_outfile_with_stdout = false;
     bool is_outfile_append = false;
     bool is_outfile_truncate = false;
-    ASTPtr format;
+    ASTPtr format_ast;
```

Comment:
I renamed `ASTQueryWithOutput::format` to `format_ast` to avoid a confusion with `IAST::format()`